REDIRECT.en.md

```diff
@@ -0,0 +1,211 @@
+# Guide to URI redirection
+
+This document provides guidance for creating and publishing
+`htaccess` files that allow redirection of semantic resources 
+URIs using the w3id.
+
```

Comment:
If this document is sourced from the w3id.org, consider linking,
so that if the w3id.org rules change, we don't need to update our docs

Reply:
The guide in `REDIRECT.en.md` (and in `REDIRECT.md`) is a sort of specialization of the [w3id guide](https://w3id.org/) for the Italia usecase. In other words, we based our guide on the w3id one, but they are different things. We can express this concept better on our redirect files.

Thanks.

Reply:
The last commit fixes this.